spec/models/user_spec.rb

```diff
@@ -11,17 +11,17 @@
     expect(@user.email).to match 'user@example.com'
   end
 
-  it "#role is :user by default" do
-    expect(@user.role).to eq :user
+  it "#role is 'user' by default" do
+    expect(@user.role).to eq "user"
   end
 
-  it "#role can be set to :vip" do
+  it "#role can be set to 'vip'" do
```

Comment:
Ah, shoot, I should have been clearer. I think these ones are okay because we _are_ setting them with symbols. My quibble with the user spec above had to do with the fact that `@user.role` returns a string.